includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,426 @@
+					'club' => esc_html__( 'Club', 'elementor' ),
+				],
+				'default' => 'spade',
+				'separator' => 'before',
+			]
+		);
+
+		$repeater->add_control(
+			'card_value',
+			[
+				'label' => esc_html__( 'Value', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'2'  => '2',
+					'3'  => '3',
+					'4'  => '4',
+					'5'  => '5',
+					'6'  => '6',
+					'7'  => '7',
+					'8'  => '8',
+					'9'  => '9',
+					'10' => '10',
+					'J'  => esc_html_x( 'J', 'Playing cards value', 'elementor' ),
+					'Q'  => esc_html_x( 'Q', 'Playing cards value', 'elementor' ),
+					'K'  => esc_html_x( 'K', 'Playing cards value', 'elementor' ),
+					'A'  => esc_html_x( 'A', 'Playing cards value', 'elementor' ),
+				],
+				'default'   => '2',
+				'separator' => 'before',
+			]
+		);
+
+		$this->add_control(
+			'cards_list',
+			[
+				'label' => esc_html__( 'Hand', 'elementor' ),
+				'type' => Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [],
+				'title_field' => '{{ card_type }} ({{ card_value }})',
+			]
+		);
+
+		$this->add_control(
+			'hover_animation',
+			[
+				'label' => esc_html__( 'Hover Animation', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'pulse' => esc_html__( 'Pulse', 'elementor' ),
+					'bounce' => esc_html__( 'Bounce', 'elementor' ),
+				],
+				'default' => 'pulse',
+				'separator' => 'before',
+			]
+		);
+
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'section_card_styles',
+			[
+				'label' => esc_html__( 'Card', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'card_background',
+			[
+				'label' => esc_html__( 'Background Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card' => '--e-playing-card-bg-color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'card_border_color',
+			[
+				'label' => esc_html__( 'Border Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card' => '--e-playing-card-border-color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'card_heart_diamond_color',
+			[
+				'label' => esc_html__( 'Hearts and Diamonds Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card-heart, {{WRAPPER}} .elementor-playing-card-diamond' => '--e-playing-card-heart-diamond-color: {{VALUE}};',
+				],
+			]
+		);
+
+		$this->add_control(
+			'card_spade_club_color',
+			[
+				'label' => esc_html__( 'Spades and Clubs Color', 'elementor' ),
+				'type' => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .elementor-playing-card-spade, {{WRAPPER}} .elementor-playing-card-club' => '--e-playing-card-spade-club-color: {{VALUE}};',
```

Comment:
```suggestion
					'{{WRAPPER}}' => '--e-playing-card-spade-club-color: {{VALUE}};',
```
You probably don't need those class names 😃